<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogdon</class>
 <widget class="QDialog" name="Dialogdon">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{border-radius: 8px; 
border: 2px solid gray;

padding: 0 8px;
background: white; }



QPushButton
{
color: gray;
font-weight: bold;
background-color: black;
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover { background-color: white;
color:black;
 text-decoration: underline; }</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>691</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tabdon">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Montant">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_time">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>31</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Event</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Mont</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>methode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete All</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Event</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mont">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="idd">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="idd_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_Methode">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Paypal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>E-Dinar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Visa</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="combo_event">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CAF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FIFA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Champions League</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>La Liga</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="event">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>FIFA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Champions League</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CAF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>La Liga</string>
    </property>
   </item>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>tabdon</zorder>
  <zorder>lineEdit_id</zorder>
  <zorder>lineEdit_Montant</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_date_time</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>id</zorder>
  <zorder>label_2</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>mont</zorder>
  <zorder>idd</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>idd_2</zorder>
  <zorder>combo_Methode</zorder>
  <zorder>combo_event</zorder>
  <zorder>event</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
